modified algob/README.md (#280)
scale-it/algo-builder: packages/algob/README.md

Algob is the CLI (Command Line Interface) for Algo Builder. Think about it as an Ethereum Truffle but for Algorand projects.

> This package provides a variety of high level tools like, `executeTransaction`, `mkTxParams`, etc. These high level tools help in making the development phase of algorand applications easier by acting as a middleware between the developer and `algosdk`.

Reviewer: ```suggestion
This package provides a variety of high level functions like, `executeTransaction`, `mkTxParams`, `Deployer`, etc. These high level functions help in making the development phase of algorand applications easier by acting as a middleware between the developer and `algosdk`.
```

Reviewer 2: Also, let's don't repeat "These high level functions ".

Author: okay

Please read the main [README](https://github.com/scale-it/algo-builder/blob/master/README.md) file for details about the project and `algob`. This file provides a development description about the `algob` package.

## How to use the API docs?

The best way to find a type, function or something related to a use case is by using the **search field** at the **top** of this page.

## Important links

+ [Home Page]()

> + [Runtime Docs]()

Reviewer: ```suggestion
+ [Runtime API Docs]()
```

Reviewer 2: Links are missing

Reviewer 2: Probably we should use here already links to newly generated docs. I will deploy new docs now.

Algob is the CLI (Command Line Interface) for Algo Builder. Think about it as an Ethereum Truffle but for Algorand projects.

This package provides a variety of high level functions like, `executeTransaction`, `mkTxParams`, `Deployer`, etc. These functions help in making the development phase of algorand applications easier by acting as a middleware between the developer and `algosdk`.

Please read the main [README](https://github.com/scale-it/algo-builder/blob/master/README.md) file for details about the project and `algob`. This file provides a development description about the `algob` package.

## How to use the API docs?

The best way to find a type, function or something related to a use case is by using the **search field** at the **top** of this page.

## Important links

+ [Home Page](https://scale-it.github.io/algo-builder)
+ [Runtime API Docs](../runtime/index)

> + [User Docs](../../guide/README)

Reviewer 3: please add `.md` and other extensions

Author: Done

Algob is the CLI (Command Line Interface) for Algo Builder. Think about it as an Ethereum Truffle but for Algorand projects.

> ## Packages

Reviewer 3: please remove this part, we are in packages folder which has sub-section `algob` and `runtime`. they both have separate readme

Author: okay

### `algob`

This package provides a variety of high level tools like, `executeTransaction`, `mkTxParams`, etc. These high level tools help in making the development phase of algorand applications easier by acting as a middleware between the developer and `algosdk`.

> ### `runtime`

Reviewer 3: runtime package is put in different folder. `packages/runtime`.

Author: As this is the main docs page right now, so I added this here. In the task description too, it is mentioned to add this. It is just a short description

This package is basically a light version of Algorand runtime. It is efficient in assisting with the tasks of processing smart contract logic i.e. TEAL files as well as in executing transactions. The package also provides various types of parsers like `addressToPk`, `stringToBytes`, `loadFromYamlFileSilent`, etc.

Please read the main [README](https://github.com/scale-it/algo-builder/blob/master/README.md) file for details about the project and `algob`. This file provides a development description about the `algob` package.

## How to use the API docs?

The best way to find a type, function or something related to a use case is by using the **search field** at the **top** of this page.

## Important links

+ [Home Page](file:///home/manofcode/scaleit/algo-builder/docs/index.html)
+ [Runtime Docs](file:///home/manofcode/scaleit/algo-builder/docs/api/runtime/index.html)

> 

Reviewer 3: ```suggestion
```

Reviewer 3: why do we have your username here ?

Reviewer 3: i don't think this link will work on other system

Author: This link will be changed while deploying. It will contain the actual address after landing page is deployed. I'll make these empty brackets.

Reviewer: No. This should have valid links. It won't "change" while deploying. You can use relative links to work. It's just that it currently won't work as master does not have generated API docs. You can update the links like:
```
[Home Page](https://scale-it.github.io/algo-builder)
[Runtime Docs](../runtime/index)
```

Reviewer: Please also add one more link for user-docs (`/guide/`)
